services.go

```diff
@@ -87,6 +87,14 @@ type RedemptionsService interface {
 	Delete(accountCode string) (*Response, error)
 }
 
+// ShippingAddressesService represents the interactions available for shipping addresses
```

Comment:
This is a really minor nit, but can you add a period to the end of this comment? Line comments end in a period, inline comments don't.